File: Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;



    public class Cliente
    {
        private int id_cliente;
        private int telefono;
        private string direccion;
        private string cuentaBancaria;

        public Cliente(int id_cliente, int telefono, string direccion, string cuentaBancaria)
        {
            this.Id_cliente = id_cliente;
            this.Telefono = telefono;
            this.Direccion = direccion;
            this.CuentaBancaria = cuentaBancaria;
        }

        public int Id_cliente{
        	set{
        		this.id_cliente = value;
			}
			get{
			return id_cliente;
			}
		}
		

		public int Telefono{
		set{
			this.telefono = value;
		}
		get{
			return telefono;
		}
		}
		

		public string Direccion{
		set{
			this.direccion = value;
		}
		get{
			return direccion;
		}
	    }

		public string CuentaBancaria{
		set{
			this.cuentaBancaria = value;
		}
		get{
			return cuentaBancaria;
		}
		}
}

File: java/Cliente.java
public abstract class Cliente{
	private int id_cliente;
	private int telefono;
	private String direccion;
	private String cuentaBancaria;

	public Cliente(int id_cliente, int telefono, String direccion, String cuentaBancaria){
		setId_cliente(id_cliente);
		setTelefono(telefono);
		setDireccion(direccion);
		setCuentaBancaria(cuentaBancaria);
	}

	public Cliente(){
	}

	public void setId_cliente(int id_cliente){
		this.id_cliente=id_cliente;
	}

	public int getId_cliente(){
	    return id_cliente;
	}
	
	public void setTelefono(int telefono){
	    this.telefono=telefono;
	}

	public int getTelefono(){
	    return telefono;
	}
	
	public void setDireccion(String direccion){
	    this.direccion=direccion;
	}
	
	public String getDireccion(){
	    return direccion;
	}

	public void setCuentaBancaria(String cuentaBancaria){
	    this.cuentaBancaria=cuentaBancaria;
	}
	
	public String getCuentaBancaria(){
	    return cuentaBancaria;
	}
	
}